.npmignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
[we already use a whitelist](https://github.com/image-size/image-size/blob/master/package.json#L6) .. this file can be removed.